addons/website_sale/static/src/js/website_sale_utils.js

```diff
@@ -32,7 +36,52 @@ function animateClone($cart, $elem, offsetTop, offsetLeft) {
     }
 }
 
+/**
+ * Allows to propose the existing products from the database.
+ *
+ * @param {ServicesMixin|Widget} self - an element capable to trigger an RPC
+ * @param {jQuery} $input
+ * @param {Object} data
+ */
+function autocompleteWithProducts(self, $input, data) {
+    var options = {
+        'display_description': data.displayDescription,
+        'display_price': data.displayPrice,
+        'display_image': data.displayImage,
+        'order_by': data.orderBy,
+    };
+    $input.autocomplete({
+        source: function (request, response) {
+            return self._rpc({
+                route: '/shop/products/autocomplete',
+                params: {
+                    'term': request.term,
+                    'limit': 7,
```

Comment:
why 7? can we make it configurable ?

Reply:
Alright, do you want the editor to allow to type any number or select a number from a list (like 1 to 15) ?

Reply:
no 0 to 16;)